src/getHumanAge.test.js

```diff
@@ -7,4 +7,25 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should convert cat and dog ages correctly', () => {
+    expect(getHumanAge(23, 23)).toEqual([1, 1]);
+  });
+
+  test('should return 0 for ages less than the first threshould', () => {
+    expect(getHumanAge(10, 10)).toEqual([0, 0]);
+  });
+
+  test('should return 1 for ages between the first and second threshould',
+    () => {
+      expect(getHumanAge(17, 17)).toEqual([1, 1]);
+    });
+
+  test('should handle large age values correctly', () => {
+    expect(getHumanAge(100, 100)).toEqual([21, 17]);
```

Comment:
The expected result for large age values is incorrect. For 100 cat years, the expected human age should be 21, and for 100 dog years, it should be 17. Update the expected result to `[21, 17]`.